aporia.nim

```diff
@@ -1485,6 +1527,21 @@ proc onSwitchTab(notebook: PNotebook, page: PNotebookPage, pageNum: guint,
   # Toggle the "Syntax Highlighting" check menu item based in the new tabs
   # syntax highlighting.
   plCheckUpdate(pageNum)
+  
+  updateMainTitle(pageNum)
+  updateStatusBar(win.tabs[pageNum].buffer)
+
+proc onAfterSwitchTab(notebook: PNotebook, page: PNotebookPage, pageNum: guint,
+                      user_data: Pgpointer) = 
+  ## After tab switch; required so the tab is visible when the check for
+  ## modification has run.
+    
+  # Check if the file changed outside the program.
+  if win.tabs[pageNum].filename != "":
+    discard checkFileUpdate(pageNum)
+  
+  updateMainTitle(pageNum)
+  updateStatusBar(win.tabs[pageNum].buffer)
```

Comment:
Necessary to have the UI updated properly. Otherwise the title and status display incorrectly when the dialog appears.